libraries/AP_Frsky_Telem/AP_Frsky_Telem.h

```diff
@@ -206,11 +206,12 @@ class AP_Frsky_Telem {
     struct
     {
         bool sport_status;
+        bool gps_refresh = false;
```

Comment:
we don't need the init of these vars, it just wastes flash

Reply:
ok bool initialization fixed